### PR TITLE
feat: add pylyzer lsp config

### PR DESCRIPTION
**Pull request by realprogrammersusevim**
Added a configuration for the pylyzer Python LSP and type checker. Kept all the default values listed in the README the same. If I didn't follow the contributing guidelines in any way please let me know so I can fix it.

### CODE REVIEW

**Comment by glepnir** [comment]
please read contribute.md and don't edit any doc file it generate from lua file.

**Comment by realprogrammersusevim** [comment]
My apologies. I misunderstood the contribute.md file when I read it and ran the docgen script locally before opening the PR. Everything should be fixed now. I undid my documentation changes with some force pushing, and made all the linters happy. With all the checks passing, is this ready to merge?

File: lua/lspconfig/server_configurations/pylyzer.lua [hunk 1]
```diff
@@ -0,0 +1,36 @@
+local util = require('lspconfig').util
+
+local bin_name = 'pylyzer'
+local cmd = { bin_name, '--server' }
+
```
Comment (glepnir) [inline lua/lspconfig/server_configurations/pylyzer.lua:5]: check windows env here. in windows you need `cmd.exe`

Comment (realprogrammersusevim) [reply lua/lspconfig/server_configurations/pylyzer.lua:5]: No clue how Windows runs executables but this seems to be how other configurations handle it and seems reasonable after reading the cmd.exe documentation.

**Pull request merged**